Tchap/Modules/Rooms/Views/RoomsInviteCell.swift

```diff
@@ -74,13 +74,13 @@ import UIKit
         }
     }
     
-    override func update(style: Style) {
-        super.update(style: style)
-        self.domainLabel.textColor = style.primarySubTextColor
-        self.leftButton.backgroundColor = style.buttonBorderedBackgroundColor
-        self.leftButton.setTitleColor(style.buttonBorderedTitleColor, for: .normal)
-        self.rightButton.backgroundColor = style.buttonBorderedBackgroundColor
-        self.rightButton.setTitleColor(style.buttonBorderedTitleColor, for: .normal)
+    override func update(theme: Theme) {
+        super.update(theme: theme)
+        self.domainLabel.textColor = theme.textTertiaryColor
+        self.leftButton.backgroundColor = theme.headerBackgroundColor
+        self.leftButton.setTitleColor(theme.headerTextPrimaryColor, for: .normal)
+        self.rightButton.backgroundColor = theme.headerBackgroundColor
+        self.rightButton.setTitleColor(theme.headerTextPrimaryColor, for: .normal)
```

Comment:
I'm not a fan to use `headerBackgroundColor`and `headerTextPrimaryColor` here, I don't want to blocck this PR on this point, but we have to create an issue to introduce correct keys in the theme for this kind of button.
This work (adding new theme keys) should be planned with Element-ios team to improve Element-ios